muxer.go

```diff
@@ -45,7 +45,7 @@ type Muxer struct {
 	buf       bytes.Buffer
 	bufWriter *astikit.BitsWriter
 
-	esContexts              map[uint16]*esContext
+	esContexts              map[uint32]*esContext
```

Comment:
Could you add the following comment on top of `esContexts`: `// We use map[uint32] instead map[uint16] as go runtime provide optimized hash functions for (u)int32/64 keys`

Reply:
Done